internal/controller/compaction/reconciler.go

```diff
@@ -38,6 +38,9 @@ import (
 const (
 	// DefaultETCDQuota is the default etcd quota.
 	DefaultETCDQuota = 8 * 1024 * 1024 * 1024 // 8Gi
+
+	// SafeToEvictKey - annotation that ignores constraints to evict a pod if set to "false".
+	SafeToEvictKey = "cluster-autoscaler.kubernetes.io/safe-to-evict"
```

Comment:
Should we use this constant which is originally defined [here](https://github.com/kubernetes/autoscaler/blob/45dfba4026a0d1bc4d5d584a57771107e1663437/cluster-autoscaler/utils/drain/drain.go#L36), by adding the Go module `k8s.io/autoscaler/cluster-autoscaler` as a dependency, instead of re-declaring the string here?

This makes the dependency on cluster autoscaler explicit in etcd-druid.
This would also help us track potential changes to this annotation since we would be forced to keep track of changes to cluster autoscaler if we depend on it.

Reply:
we don't directly consume autoscaler, we use forked of autoscaler https://github.com/gardener/autoscaler.
I don't think it's good to create dependency just for 1 constant. Even g/g is not using const via autoscaler: https://github.com/gardener/gardener/blob/8b7e4cc9e29a0bae4201ce118f45db9d08b721e4/pkg/component/etcd/etcd/etcd.go#L224

Reply:
I don't particularly feel strongly about this.
We shouldn't ponder too much and get this in for now. We can always add the dependency later on if we feel that is the better way of doing it.